en/config_vtol/vtol-back-transition-tuning.md

```diff
@@ -0,0 +1,36 @@
+# VTOL back transition tuning
+
+> **Tip** Some of the following features will be available in PX4 version 1.7 and are currently only available on the experimental development branch.
+
+When a VTOL performs a back transition (transition from fixed wing mode to multicopter) it needs to slow down before the multicopter can take proper control.
+As of PX4 version 1.7, and on the current developer branch, the vehicle will consider the back transition complete when the horizontal speed has reached multicopter cruise speed (MPC_XY_CRUISE) or when the back transition duration (VT_B_TRANS_DUR) has passed (which ever comes first).
+
+## Back transition duration
```

Comment:
For sub-titles we use "first letter capitalisation" exactly as you have done. This rule is not applied consistently though.

In this case "Back-transition duration" (with the hyphen)